README.md

```diff
@@ -196,8 +196,8 @@ or you just untarred the Redis distribution tar ball. In both the cases
 you are basically one step away from the source code, so here we explain
 the Redis source code layout, what is in each file as a general idea, the
 most important functions and structures inside the Redis server and so forth.
-We keep all the discussion at an high level without digging into the details
-since this document would be huge otherwise, and our code base changes
+We keep all the discussion at a high level without digging into the details
```

Comment:
This sentence is long and feel overly heavy. Perhaps something along these lines:
"Keeping the concepts in this document at a high level prevents this document from becoming overly large. General ideas are fine as they provide a starting point for greater understanding. Moreover, most of the code ..."